hyper/http20/connection.py

```diff
@@ -482,6 +497,9 @@ def _send_cb(self, frame, tolerate_peer_gone=False):
 
         data = frame.serialize()
 
+        if frame.body_len > FRAME_MAX_LEN:  # pragma: no cover
```

Comment:
So this is the point where we need to abort based on the set max frame size. Can you change this check to do that?